content/logs/explore.md

```diff
@@ -84,14 +84,6 @@ Once this is done, the value of this attribute is stored **for all new logs** an
 Use facets to easily filters on your logs. The search bar and url automatically reflect your selections.
 {{< img src="logs/explore/facet_panel.png" alt="Facet panel" responsive="true" popup="true">}}
 
-### Expanded view
```

Comment:
We should replace this by a section on the multiline display

Reply:
👌